.gitignore

```diff
@@ -28,6 +28,7 @@ doc/_build/
 doc/gallery/
 doc/projections/
 doc/tutorials/
+doc/starter-tutorial/
```

Comment:
Do you think it's better to rename `starter-tutorial` to `get-started`? I think `get-started` is more commonly used.

Reply:
I'm fine with either way. I'll change it unless anyone has any issues with it?

Reply:
The heading for the 'First figure' tutorial is called 'Getting started' :sweat_smile: How about `quickstart` or `first-map`/`first-figure`?

Other 'getting-started' pages for reference:
- [`xarray`](https://xarray.pydata.org/en/stable/getting-started-guide/index.html)
- [`geopandas`](https://geopandas.org/en/stable/getting_started.html)
- [`pandas`](https://pandas.pydata.org/docs/getting_started/index.html)
- etc

Reply:
I think both `quickstart` and `first-map` make it seem like it is a one-page guide to install PyGMT and make a quick figure. I'm envisioning this tutorial as a series of pages that create different figures and progressively builds upon previous concepts.